src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -72,10 +73,10 @@ public static Address parseAddress(String address) throws ParseException {
     public static Tag parseTag(String tag) throws ParseException {
         requireNonNull(tag);
         String trimmedTag = tag.trim();
-        if (!Tag.isValidTagName(trimmedTag)) {
+        if (!Tag.isValidTag(trimmedTag)) {
             throw new ParseException(Tag.MESSAGE_CONSTRAINTS);
         }
-        return new Tag(trimmedTag);
+        return Tag.create(trimmedTag);
```

Comment:
Reason for creating a static create tag method rather than returning a new tag object

Reply:
to check if there's duplicate Tags